js/class/ToDoApplication.js

```diff
@@ -0,0 +1,136 @@
+class ToDoApplication{
+    constructor(_dom){
+        this.toDo = new ToDo();
+        this.init(_dom);
+    }
+    init(_dom){
+        this.loadTodos(this.toDo.items);
+        this.newTodo(_dom);
+    }    
+    newTodo(_dom){
+        const input = _dom.querySelector('.new-todo');
+        input.addEventListener('keydown', (e) =>{
+            let keyCode = e.keyCode;
+            let todoText = e.target.value;
+            keyCode == 13 ? (this.addTodo(todoText), e.target.value = '') : false;
+        });
+    }
+    addTodo(todoText){
+        const newItem = this.toDo.addItem(todoText);
+        this.appendItem(newItem);
+        this.deleteTodoItem();
+        this.setComplete();
+        this.countToDos();
+    }
+    loadTodos(toDoItems){
+        toDoItems.forEach(this.appendItem);
+        this.editingToDos();
+        this.deleteTodoItem();
+        this.setComplete();
+        this.countToDos();
+        this.filterToDos();
+        
+    }
+    appendItem(newItem){
+        const ulList = document.querySelector('#todo-list');
+        let liTemplate = `<li id="${newItem.id}" class="${newItem.isCompleted ? 'completed' : ''}">
+                        <div class="view">
+                        <input class="toggle" type="checkbox" ${newItem.isCompleted ? "checked" : ''} />
+                        <label class="label">${newItem.text}</label>
+                        <button class="destroy"></button>
+                        </div>
+                        <input class="edit" value="${newItem.text}" />
+                    </li>`;
+        ulList.innerHTML += liTemplate;
+    }
+    deleteTodoItem(){
+        let list = document.querySelector('#todo-list');
+        let btn = document.querySelectorAll('.destroy');
+        let parent,liId;
+        for(let i =0; i < btn.length; i++){
+            btn[i].onclick = (e) => {
+                parent = e.target.parentNode.parentNode;
+                liId = parent.id;
+                list.removeChild(parent);
+                this.toDo.deleteItem(liId);
+                this.countToDos();
+            };
+        }
+        
+    }
+    setComplete(){
+        let checkBoxList = document.querySelectorAll('.toggle');
+        checkBoxList.forEach((check) => {
+            check.addEventListener('change',(e) => {
+                let active = e.target.parentNode.parentNode;
+                let activeId = active.id;
+                if(e.target.checked){
+                    e.target.setAttribute('checked','checked');
+                    active.classList.value = 'completed';
+                    this.toDo.isCompleted(activeId,true);
+                    this.filterToDos();
+                }else{
+                    e.target.setAttribute('checked',false);
+                    active.classList.value = '';
+                    this.toDo.isCompleted(activeId,false);
+                    this.filterToDos();
+                }
+            });
+
+        });
+    }
```

Comment:
```suggestion
    setComplete(){
        document.querySelectorAll('.toggle').forEach(check => {
            check.addEventListener('change', ({ target }) => {
                const $todoItem = target.closest('li');
                this.toDo.isCompleted($todoItem.id, target.checked);
                this.loadTodos(this.toDo.items);
            });
        });
    }
```
여기도 이런식으로 loadTodos를 사용하면 될 것 같습니다!